core/src/main/java/fr/sncf/osrd/envelope/package-info.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * <h1>Envelopes</h1>
+ * <h2>Overview</h2>
+ * <p>Envelopes are a framework for working with position / speed curves, with constant acceleration between each point.
+ * This isn't as simple as it may seem, as acceleration works over time, and the envelopes work is over space.</p>
+ *
```

Comment:
Envelopes are a collection of envelope parts. An envelope part is a continuous space-speed curve, with monotonically increasing position. It can be seen as a sequence of steps, each step taking a given time, moving forward in space, and having a start and an end speed. Each envelope part starts where the previous one ends, resulting in a space-continuous envelope.